Makefile

```
@@ -115,6 +115,11 @@ check_modules: $(BASE_I) $(MODULES_C)
 
 document: $(ALL_PKGS_D) .doc/base/all
 
+pkgdocs:
+	Rscript scripts/build_pkgdown.R $(ALL_PKGS) || exit 1
+	
+.PHONY: all install check test document shiny pkgdocs
```

1. I'm on the fence about whether pkgdown build should happen here or in the `book_source/` build (which is currently separate from this Makefile). The fact this is where we already have a list of all the packages certainly helps make the case for doing it here.
2. If building here, add `base/all` to the target list (confusingly, it's not part of `$ALL_PKGS`) 
3. there's already a .PHONY declaration on line 106, so please add `pkgdocs` to that instead of making a new one here
```suggestion
pkgdocs:
	Rscript scripts/build_pkgdown.R $(ALL_PKGS) base/all || exit 1
	
```